### DISCUSSION TITLE
Is it possible for custom sign in form to be a component instead of a page?

### DISCUSSION THREAD

**Discussion by djvid** [nextauthjs/next-auth#1426] [Help] [answered]
When user hover login button, the sign in form appears as a drop down. How can I implement this feature instead of redirecting to sign in page?

**Comment by StefanSelfTaught** [comment, marked as answer]
This will not redirect to signIn page and if you pass `redirect: false`  it won't redirect after login.
```js
signIn('credentials', { username: 'jsmith', password: '1234', redirect: false })
```
See more here: https://next-auth.js.org/getting-started/client#signin

**Reply by djvid** [reply]
Hi Stefan, thanks for your reply I'm able to start the email sign in flow from my drop down. But what is the purpose of 'redirect: false', the behaviour is same whether i pass it or not. 

After signing in with email, its redirecting me to this page but instead I would like the user to remain in the same page and just display a simple toast to let user know that verification is sent. Is it possible?

![image](https://user-images.githubusercontent.com/61406465/109955879-f0a80980-7d1d-11eb-99fd-34bf9981d421.png)

**Reply by StefanSelfTaught** [reply]
Make sure you are using the latest version of NextAuth, 3.7.0.

**Reply by djvid** [reply]
Thank you so much Stefan! Its working now 👍
